rpc/rpc-lib/src/rpc-clnt.c

```diff
@@ -23,14 +23,14 @@ void
 rpc_clnt_reply_deinit(struct rpc_req *req, struct mem_pool *pool);
 
 struct saved_frame *
-__saved_frames_get_timedout(struct saved_frames *frames, uint32_t timeout,
-                            struct timeval *current)
+__saved_frames_get_timedout(struct saved_frames *frames, time_t timeout,
```

Comment:
if you are already changing the signature, please make the function static.

Reply:
OK